src/session.rs

```diff
@@ -220,7 +220,7 @@ impl Inner {
                     if let Some(cookie) = jar.signed(&self.key).get(&self.name) {
                         let value = cookie.value().to_owned();
                         let cachekey = (self.cache_keygen)(&cookie.value());
-                        return Either::Left(
+                        return 
                             self.addr.send(Command(resp_array!["GET", cachekey])).map(
```

Comment:
There is no need in this map in async/await context

Reply:
> I don’t understand purpose of this pr. If you change to async why you keep future combinators?

Yeah.. I see what you mean.  I managed to remove all the map combinators and consequently the need for FutureExt, too, in latest commit.